albumentations/core/utils.py

```diff
@@ -48,8 +48,9 @@ def postprocess(self, data):
         rows, cols = data["image"].shape[:2]
 
         for data_name in self.data_fields:
-            data[data_name] = self.filter(data[data_name], rows, cols)
-            data[data_name] = self.check_and_convert(data[data_name], rows, cols, direction="from")
+            if data_name in data:
+                data[data_name] = self.filter(data[data_name], rows, cols)
+                data[data_name] = self.check_and_convert(data[data_name], rows, cols, direction="from")
```

Comment:
I agree with this changes if you will add warning message:
```python
if data_name in data:
    ...
else:
    warnings.warn("Can not find {} in provided data.".format(data_name), RuntimeWarning)
```

Reply:
This warning will be desplayed every time augmentation is called. In most applications it is not 1 or 2 times :). What for? I can't imagine use case when a didn't add some parameter and will not see it without such a warning. Not to mention the fact that warnings not always are noticed and correctly interpreted.
An only case when it can be usefull is when I misspelled parameter name. Now such a parameter data is just passed through without processing. And it is quite another case because this code even will not be accessed, if no pre- and post- processing is required.
For example:
```
aug = Compose([HorizontalFlip(), RandomCrop(100,100)])  # no processors are created
res = aug(img, bbooks=boxes)  # misspelled
img, boxes = res["image"], res["bbooks"]  # bbooks copy-pasted from above
```
- produce no error or warnings but results in quite unexpected way.
I suppose an exeption should be generated in such case. If somebody needs extra data `bbooks` to be passed through without modification, it should be explicitly defined through `additional_targets` argument, that should supoport not only bbox and keypoint, but also targets with no processing.
But in my oppinion it is another issue.